h/models/annotation.py

```diff
@@ -12,6 +12,58 @@
 from h.util.user import split_user
 
 
+class AnnotationSlim(Base):
```

Comment:
I was not planning to add the other metadata table I'd call this one 

`AnnototationMetadata`

we could call my `AnnotationMedata` AnnotationExtra or similar but annotation.extra is already a thing.

Slim is best I could come up with.